packages/client/webapp/src/stores/auth.ts

```diff
@@ -0,0 +1,55 @@
+import { defineStore } from 'pinia';
```

Comment:
Why `pinia` instead of the web api LocaleStorage?

Reply:
localstorage could be used as well but figure the app will need state management so might as well use it for this as well. works well with vue and wont accidentally forget to clear the tokens

Reply:
I see, I wasn't really keen on using a JS framework specific state management dependency, because people will start to use to bypass simple event/prop communication path, and do weird thing.

But let's move on, and I will keep an eye on it.